c2cgeoportal/scaffolds/create/+package+/static-ngeo/js/components/README.md_tmpl

```diff
@@ -0,0 +1,11 @@
+Directives and their partials are structured by components
+
+A subdirectory is created for each component.
+The component partials are directly stored in the subdirectory.
+
+The partials are loaded individually at runtime in debug mode but preloaded in a template cache for production.
+For this mechanism to work correctly the directive template URL must follow some conventions:
+
+In the directive, the template url is written as follow: "templateUrl: {{package}}.baseTemplateUrl + '/<component>/<partial>.html'".
+
+In the main html file, the debug section should contain "{{package}}.templateBaseUrl = '${request.static_url("{{package}}:static-ngeo")}';"
```

Comment:
Shouldn't be also added here?
https://github.com/camptocamp/c2cgeoportal/blob/aa4568be45b4c8694b70610cb71b7ce8b4c1d554/c2cgeoportal/scripts/import_ngeo_apps.py#L229